zerver/views/streams.py

```diff
@@ -346,10 +346,13 @@ def add_subscriptions_backend(request, user_profile,
         else:
             msg = ("Hi there!  %s just created a new stream '%s'. %s"
                        % (user_profile.full_name, created_streams[0].name, stream_button(created_streams[0].name)))
+            msg_self = ("Hi there!  You just created a new stream '%s'. %s"
+                       % (created_streams[0].name, stream_button(created_streams[0].name)))
             for realm_user_dict in get_active_user_dicts_in_realm(user_profile.realm):
-                # Don't announce to yourself or to people you explicitly added
-                # (who will get the notification above instead).
-                if realm_user_dict['email'] in principals or realm_user_dict['email'] == user_profile.email:
+                if realm_user_dict['email'] == user_profile.email:
+                    notifications.append(internal_prep_message(settings.NOTIFICATION_BOT,
+                                                           "private",
+                                                           realm_user_dict['email'], "", msg_self))
```

Comment:
I think this breaks the previous case where `realm_user_dict['email'] in principals` since that will no longer `continue`?